### PR TITLE
bump symfony version to 7

### PR DESCRIPTION
**Pull request by abdellahrk**
## Add support for Symfony 7

This merely adds supports for Symfony 7 which has just been released.

## Changelog

```markdown
### Added
- Support for Symfony 7
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
     ],
     "homepage": "https://docs.sonata-project.org/projects/SonataSeoBundle",
     "require": {
-        "php": "^8.0",
+        "php": "^8.0|^8.2",
```
Comment (VincentLanglet) [inline composer.json]:
```suggestion
        "php": "^8.0",
```
Is ok, it means already 8.0, 8.1, 8.2, 8.3 etc

File: composer.json [hunk 2]
```diff
@@ -20,17 +20,17 @@
...
         "sonata-project/block-bundle": "^4.18 || ^5.0",
         "sonata-project/exporter": "^2.0 || ^3.0",
-        "symfony/config": "^5.4 || ^6.2",
-        "symfony/dependency-injection": "^5.4 || ^6.2",
-        "symfony/form": "^5.4 || ^6.2",
-        "symfony/framework-bundle": "^5.4 || ^6.2",
-        "symfony/http-foundation": "^5.4 || ^6.2",
-        "symfony/http-kernel": "^5.4 || ^6.2",
-        "symfony/options-resolver": "^5.4 || ^6.2",
+        "symfony/config": "^5.4|^6.2|^7.0",
```
Comment (VincentLanglet) [inline composer.json]: Please keep the same format that before
```
"^5.4 || ^6.2 || ^7.0",
```

**Review by VincentLanglet — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (abdellahrk) [reply composer.json]: Okay @VincentLanglet noted.

File: composer.json (hunk 2 above)
Comment (abdellahrk) [reply composer.json]: fixed @VincentLanglet

**Comment by VincentLanglet** [comment]
The rector issue is unrelated, I'm on it

**Review by VincentLanglet — DISMISSED** [review]

**Comment by VincentLanglet** [comment]
Thanks, I'll merge this and release it when I have time. Unless @jordisala1991 do it

**Comment by VincentLanglet** [comment]
Sorry, do you have time to rebase the branch ? @abdellahrk

**Comment by abdellahrk** [comment]
Yes sure @VincentLanglet

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**